src/components/StateBar/index.tsx

```diff
@@ -74,19 +70,19 @@ export const Bar: VoidFunctionComponent<StateBarBarProps> = ({
   )
 }
 
-Bar.propTypes = {
+StateBarBar.propTypes = {
   unlimited: PropTypes.bool,
   value: PropTypes.number,
 }
 
 type StateBarType = typeof Box & {
-  Bar: typeof Bar
-  State: typeof State
+  Bar: (props: StateBarBarProps) => JSX.Element
```

Comment:
Why not leave the typeof ?

Reply:
Just though that was more cleaner if we change Bar type it ensure that we have a JSX.Element at the end 🤔